test/common/config/api_type_oracle_test.cc

```diff
@@ -12,6 +12,10 @@ namespace Config {
 namespace {
 
 TEST(ApiTypeOracleTest, All) {
+  // For proto descriptors only
+  static_cast<void>(envoy::config::filter::http::ip_tagging::v2::IPTagging::RequestType());
```

Comment:
This change and the BUILD fixes are not strictly part of the tooling, but they are nop changes to the rest of the tree that are needed to test the tool; they fix missing deps and linking requirements.